web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -762,10 +768,14 @@ $.extend(fixmystreet.set_up, {
         if ($('#map_permalink').length === 0) {
             $('#sub_map_links').append('<a href="#" id="map_permalink">' + translation_strings.permalink + '</a>');
         }
+        if ($('#map_filter').length === 0) {
+            $('#sub_map_links').prepend('<a href="#side" id="map_filter" class="hidden">Filter</a>');
+        }
     }
 
     if ($('.mobile').length) {
         $('#map_permalink').addClass('hidden');
+        $('#map_filter').removeClass('hidden');
```

Comment:
Similarly, this then shows it on mobile pages where it doesn't work properly.
I think it needs to do all its things in apply/remove_ui.